docs/source/kedro_project_setup/configuration.md

```diff
@@ -320,8 +320,8 @@ credentials = conf_loader.get("credentials*", "credentials*/**")
 
 This will load configuration files from `conf/base` and `conf/local` that have filename starting with `credentials` or are located inside a folder with name starting with `credentials`.
 
-```eval_rst
-.. note::  Since ``local`` is set as the environment, the configuration path ``conf/local`` takes precedence in the example above. Hence any overlapping top-level keys from ``conf/base`` will be overwritten by the ones from ``conf/local``.
+```{note}
+Since ``local`` is set as the environment, the configuration path ``conf/local`` takes precedence in the example above. Hence any overlapping top-level keys from ``conf/base`` will be overwritten by the ones from ``conf/local``.
```

Comment:
```suggestion
Since `local` is set as the environment, the configuration path `conf/local` takes precedence in the example above. Hence any overlapping top-level keys from `conf/base` will be overwritten by the ones from `conf/local`.
```